<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardNewClassification</class>
 <widget class="QWizard" name="WizardNewClassification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Classification Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Configurate Classifier</string>
   </property>
   <property name="subTitle">
    <string>Configurate the param of classifier and classification</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>General</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input layer</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxInput</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBoxAssessment">
         <property name="text">
          <string>Assessment</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Output Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEditOutputName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxSample"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sample</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxSample</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxInput"/>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxAssessment"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditOutputName">
         <property name="text">
          <string>New Classification</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxParam">
      <property name="title">
       <string>Classification Params</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Classifier</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxClassifier</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxClassifier">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPropertyEditorWidget" name="propertyEditWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Configurate Classifier</string>
   </property>
   <property name="subTitle">
    <string>Configurate the param of classifier and classification</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Data</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxNormalize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Normalize</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxNormalize">
           <item>
            <property name="text">
             <string notr="true">stddev</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">calculas</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">rank</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxPCA">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>PCA</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxPCA">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QTreeWidget" name="treeWidgetAllFeatures">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QListView" name="listViewSelectedFeatures"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>All Feature Attributes:</string>
           </property>
           <property name="buddy">
            <cstring>treeWidgetAllFeatures</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Selected Feature Attributes:</string>
           </property>
           <property name="buddy">
            <cstring>listViewSelectedFeatures</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxPreview">
      <property name="title">
       <string>Preview</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButtonOriginal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Original</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonTransformed">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Transformed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CDTHistogramPlot" name="qwtPlot">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDTHistogramPlot</class>
   <extends>QFrame</extends>
   <header>cdthistogramplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QPropertyEditorWidget</class>
   <extends>QTreeView</extends>
   <header>QPropertyEditorWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
